ietf/templates/doc/document_info.html

```diff
@@ -59,16 +54,6 @@
                         {% if submission %}({{ submission|safe }}){% endif %}
```

Comment:
We should look to make sure, but off the cuff, I don't think submission will be (or at least should be) non None if draft.type == "rfc"

Reply:
Sorry I don't know how to interpret the code changes for this. Are you suggesting that the code should look like,
```
{% if doc.type_id == "rfc" %}({{ submission|safe }}){% else %}something else?{% endif %}
```

Reply:
well, it would have been the other way around - the submission type really applies to documents of type `draft` not of type `rfc`. I read through the code and you can leave this line as is.

That said, what the code is doing is questionable (this PR shouldn't change it, but I'll capture things here to share knowledge): document_main has a conditional branch for documents of type `rfc` that renders document_rfc.html with `submission` in its context. It computes the value for `submission` here: https://github.com/ietf-tools/datatracker/blob/ff8898186ba06072d0e226a6c88bb6e371cc3dee/ietf/doc/views_doc.py#L316-L329 using the RFCs group instead of the draft the RFC came from. This value is rendered next to the draft name if the document came from a draft. For example, for RFC8989 it's the word "individual" at:
![image](https://github.com/user-attachments/assets/5b77e85e-2445-46f6-8084-548aace43090)

Notice that for an April 1 RFC (which does not come from a draft), the `submission` value is not rendered at all. (See RF9564).

What should really happen is that submission should be calculated from the draft the rfc came from. It is likely at the moment that the rfc _always_ inherits the information used to calculate `submission` from the draft with no change - but someday that may not be true.